include/EASTL/InsertionSortDriver.cpp

```diff
@@ -0,0 +1,208 @@
+#include <stdio.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <time.h>
```

Comment:
I think we should migrate something like to this the EASTL benchmarks instead of a completely separate driver.